sydent/threepid/bind.py

```diff
@@ -40,13 +40,30 @@
 
 logger = logging.getLogger(__name__)
 
+def parseMxid(mxid):
+    if len(mxid) > 255:
+        raise Exception("This mxid is too long")
+
+    if len(mxid) == 0 or mxid[0:1] != "@":
+        raise Exception("mxid does not start with '@'")
+
+    parts = mxid[1:].split(':', 1)
+    if len(parts) != 2:
+        raise Exception("Not enough colons in mxid")
+
+    return parts
+
+class BindingNotPermittedExceotion(Exception):
```

Comment:
s/Exceotion/Exception/

Reply:
oops. Also I managed to lose the code that caught it somehow.